docs/definition.rst

```diff
@@ -1,27 +1,31 @@
-Execution Environment Definition
-================================
-
-The execution environment (EE) definition file supports multiple versions.
+.. _builder_ee_definition:
 
-  * Version 1: Supported by all ``ansible-builder`` versions.
-  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and later.
-  * Version 3: Supported by ``ansible-builder`` versions ``3.0`` and later.
+Execution environment definition
+================================
 
-If the EE file does not specify a version, version 1 will be assumed.
+You define the content of your execution environment in a YAML file. By default, this file is called ``execution_environment.yml``. This file tells Ansible Builder how to create the build instruction file (Containerfile for Podman, Dockerfile for Docker) and build context for your container image.
 
 .. note::
+   This page documents the definition schema for Ansible Builder 3.x. If you are running an older version of Ansible Builder, you need an older schema version. Please consult older versions of the docs for more information. We recommend using version 3, which offers substantially more configurability and functionality than previous versions.
 
-    This version of the documentation discusses only the latest format version.
-    For further details on older formats, reference previous versions of the
-    documentation.
+.. contents::
+   :local:
 
-Version 3 Format
-----------------
+Overview
+--------
+The Ansible Builder 3.x execution environment definition file accepts seven top-level sections:
+* additional_build_files
+* additional_build_steps
+* build_arg_defaults
+* dependencies
+* images
+* options
+* version
```

Comment:
This doesn't display correctly.

FYI, the `docs/readthedocs.org:ansible-builder` job provides a preview of what the resulting documentation will look like (click `Details` next to the job).

Reply:
ah, needs a blank line, probably